src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java

```diff
@@ -112,6 +115,25 @@ public ResponseEntity<Page<AdvertisementTitleDto>> getPageOfAdvertisementsByKeyw
                 new ResponseEntity<>(allByKeyword, HttpStatus.OK);
     }
 
+    @GetMapping(ApiKey.ADV_SEARCH_PAGINATED_BY_CATEGORY_ID)
+    @Operation(summary = "Find advertisements by category id")
```

Comment:
It'd be better to update the description. The valid behavior is finding advertisements having the same category.

Reply:
refacted